bftengine/include/bftengine/SimpleClient.hpp

```diff
@@ -45,7 +45,25 @@ enum ClientMsgFlag : uint8_t {
   EMPTY_CLIENT_REQ = 0x10
 };
 
-enum OperationResult : int8_t { SUCCESS, NOT_READY, TIMEOUT, BUFFER_TOO_SMALL };
+enum OperationResult : int8_t { SUCCESS, NOT_READY, TIMEOUT, BUFFER_TOO_SMALL, INVALID_REQUEST };
+
+struct ClientRequest {
```

Comment:
I did not understand the question.

Reply:
ClientRequestMsg and ClientRequestMsgHeader. I think I realized why you did this though and it's fine. it's because of changes to communication layer. Also, as you say it's just in the SimpleClient which is going away. Carry on :)